scrapers/DesperateAmateurs.yml

```
@@ -0,0 +1,35 @@
+name: "DesperateAmateurs"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - desperateamateurs.com/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="title_bar"]/text()
+      Details: //div[@class="gallery_description"]/text()
+      Performers:
+        Name: //a[@class="model_category_link"][contains(@href, 'sets.php')]/text()
+      Tags:
+        Name: //a[@class="model_category_link"][contains(@href, 'category.php')]/text()
+      Image:
+        selector: //img[@class="thumbs"]/@src
+        postProcess:
+          - replace:
+            - regex: (.+image:\s+")(.+jpg)(.+)
+              with: $2
+            - regex: ^
+              with: "https://desperateamateurs.com/"
+      Studio:
+        Name:
+          fixed: Desperate Amateurs
+      Date:
+        selector: //td[@class="date"]/text()
```

same issue with the date
the selector matches the dates from the relevant clips
```suggestion
        selector: //table[@width="100%"]//td[@class="date"]/text()
```
Only matches the correct one